### ISSUE TITLE
Support "Owned" Avatars

### ISSUE THREAD

**Issue by tomasienrbc** [gitcoinco/ethavatar#1]
Avatars that are crypto-collectibles the wallet "owns" would be a fantastic use case that would require some light modification. 

Ideally two things are different: 

- The avatar is represented visually by the collectible, but in code by the asset address
- ETHAvatar UI layers have a way to understand and visualize that the address "owns" their avatar

The initial use case would be to let Cryptopunks owners use their Cryptopunks as their ETHAvatar and have them receive an additional layer of UI and network context for having done so. 

Would love feedback on how this could/should work

**Comment by tarekskr** [comment]
I think what you're suggesting is to allow users to set an arbitrary ERC-721 token that they own as their ETH Avatar, correct? Interesting!

**Comment by tomasienrbc** [comment]
@tarekskr that's right! Cryptopunks aren't pure ERC721 but supporting any arbitrary ERC721 to start would work too. Add other standards or non-standard assets as needed!

**Comment by owocki** [comment]
this is such a good idea!
